Sources/UIViewExtensions.swift

```diff
@@ -31,8 +31,8 @@ extension UIView {
 
     /// EZSwiftExtensions, add multiple subviews
     public func addSubviews(_ views: [UIView]) {
-        views.forEach { eachView in
-            self.addSubview(eachView)
+        views.forEach { [weak self] eachView in
```

Comment:
`forEach` is not `@escaping` method.  So i think this case is not need to describe `weak self`.

`public func forEach(_ body: (Element) throws -> Void) rethrows`